test/uk/gov/hmrc/trusts/services/OtherIndividualTransformationServiceSpec.scala

```diff
@@ -86,6 +87,60 @@ class OtherIndividualTransformationServiceSpec extends FreeSpec with MockitoSuga
             "internalId", RemoveOtherIndividualsTransform(0, otherIndividual, LocalDate.of(2013, 2, 20)))
         }
       }
+
+      "for amend other individual using the transformation service" in {
+
+        val transformationService = mock[TransformationService]
+        val service = new OtherIndividualTransformationService(transformationService, LocalDateMock)
+
+        val amendedOtherIndividual = NaturalPersonType(
+          name = NameType("First", None, "Last"),
+          dateOfBirth = None,
+          identification = None,
+          lineNo = None,
+          bpMatchStatus = None,
+          entityStart = LocalDate.now(),
```

Comment:
I would use the local date mock, and preferably not use LocalDate.now even there.